velox/exec/GroupingSet.h

```diff
@@ -128,7 +128,7 @@ class GroupingSet {
   /// from non-spilled partitions, then merges spill runs and
   /// unspilled data form spilled partitions. Returns nullptr when at
   /// end.
-  bool getOutputWithSpill(const RowVectorPtr& result);
+  bool getOutputWithSpill(int32_t batchSize, const RowVectorPtr& result);
```

Comment:
nit: perhaps, update the comment to explain how batchSize is used

Reply:
Thanks. Yeah, actually the current semantics is not consistent with other parts. I updated the code logic.